pkg/kubelet/cm/.import-restrictions

```diff
@@ -0,0 +1,9 @@
+rules:
+  # stop unnecessary usage of runc vendored code in kubelet
+  - selectorRegexp: "github[.]com/opencontainers/runc"
+    allowedPrefixes:
+      - "github.com/opencontainers/runc/libcontainer/configs"
+      - "github.com/opencontainers/runc/libcontainer/apparmor"
+      - "github.com/opencontainers/runc/libcontainer/cgroups"
+      - "github.com/opencontainers/runc/libcontainer/userns"
+      - "github.com/opencontainers/runc/libcontainer/utils"
```

Comment:
if we're still allowing all of these, are we actually effectively protecting against increasing use?